### DISCUSSION TITLE
Menu not working

### DISCUSSION THREAD

**Discussion by DCphat6** [nurlan-aliyev/RestaurantManagementSystem#2] [Q&A]
Has anyone downloaded this code and gotten it to work on raspbery pi?

The only thing that works for me is the about item, all others do nothing

**Comment by nurlan-aliyev** [comment]
It uses SQLite, and this app is made for PC in mind, if you are interested you can change it to RP as you wish

**Comment by era1122222** [comment]
hey the kitchen after i add products , is not working how can i do to work the kitchen after i add products
